composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "salcode/bootstrap-genesis",
+    "description": "WordPress Genesis Child Theme setup to use Bootstrap, Sass, and Grunt",
+    "keywords": ["wordpress", "theme"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Sal Ferrarello",
+            "email": "sal@ironcodestudio.com",
+            "homepage": "http://salferrarello.com/"
+        }
+    ],
+    "autoload": {
```

Comment:
@dosten I'm not clear on the purpose of this autoload line in this `composer.json` file.  I think we could get rid of it but if there is a reason to keep it, I'd love to learn about it.